src/pages/Homepage/LendByCategoryHomepage.vue

```diff
@@ -12,6 +12,18 @@
 					<p class="featured-loans__body">
 						Kiva empowers underserved people to achieve their
 						dreams by crowdfunding loans and unlocking capital.
+						<a
+							class="show-for-large featured-loans__body--link"
```

Comment:
Here the link isn't a modifier but instead another element of the block featured-loans. So it shouldn't have two dashes which are reserved for modifiers. A class like `featured-loans__cta`, or `featured-loans__body-link` would be more apt.

Reply:
I believe this should be a <router-link> instead of <a>